map/framework.hpp

```diff
@@ -591,6 +591,10 @@ class Framework
   bool IsOnRoute() const { return m_routingSession.IsOnRoute(); }
   bool IsRouteNavigable() const { return m_routingSession.IsNavigable(); }
   void BuildRoute(m2::PointD const & start, m2::PointD const & finish, uint32_t timeoutSec);
+  /// There were a bug inside FollowRoute method. Router follows route even it wasn't call.
+  /// So we deside to add special method to disable following mode, because we dont want to break
+  /// behavior users familiar to.
```

Comment:
FollowRoute had a bug where the router followed the route even if the method hadn't been called.
This method was added because we do not want to break the behaviour that is familiar to our users.

Сейчас этого бага нет?